### ISSUE TITLE
Docs for non-cli usage

### ISSUE THREAD

**Issue by shatz01** [Lightning-AI/litgpt#1310]
Thanks for the great lib!

My only gripe is that the documentation and tutorials only show how to use this lib as a cli. I want to do some very custom stuff in the future so id like to use it as python code. 

Are there any tutorials for doing so? If not, may I know why its only meant to be used as a CLI?

**Comment by williamFalcon** [comment]
good point! @lantiga let’s add this to our queue. i agree docs on how to do your own pretraining, etc are useful

**Comment by shatz01** [comment]
Yes! Happy to help out! I can make some beginner notebooks or something. Any direction for where to start? @williamFalcon 

Ive made a few small contributions to lightning in the past :)

**Comment by shatz01** [comment]
Hey @williamFalcon  @lantiga 
Ive made exactly the code I was looking for in https://github.com/Lightning-AI/litgpt/pull/1331 :)
